packages/ra-core/src/controller/field/useReferenceArrayFieldController.ts

```diff
@@ -113,8 +125,13 @@ export const useReferenceArrayFieldController = <
 
     return {
         ...listProps,
+        canAccess,
+        // When canAccess is false, isPending will always be true as the underlying query is not enabled
```

Comment:
This is concerning. An unauthorized component shouldn't be "pending". The same applies to other components that disable data provider calls depending on the access control.

Reply:
- when `canAccessPending` is true, `isPending` is `true`
- when `canAccess` is `false`, `isPending` is `false`
- when `canAccess` is `true` and `isPending` is `true`, `isPending` is `true`

I don't see your issue here